### ISSUE TITLE
Document approaches for users setting primary event priors and reasons to do so.

### ISSUE THREAD

**Issue by seabbs** [epinowcast/epidist#481]
From @athowes in #474: Is there some way we can document how users can make use of the priors on window functionality? And when they would do it? Inclusion in some vignette? Create new issue on this? I think otherwise it is quite a challenging thing to figure out.

This issue is blocked by #221

**Comment by athowes** [comment]
Would this make sense to be included in a hypothetical vignette about use of the latent model #486? It seems possible that the primary event priors are of greatest importance in a low data setting (as with the latent model).

**Comment by seabbs** [comment]
I think they are of most importance when they are wide or there is a very high growth rate (or both) so probably need standalone docs
